src/client/components/ResultMessageModalWindow/FinalMessage.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './ResultMessageModalWindow.styles.css';
+import PropTypes from 'prop-types';
+
+export default function FinalTaskMessage({ correctAnswer, checkpoints }) {
+  if (correctAnswer > 3) {
+    return (
+      <div className="final_message-component">
+        <p>
+          You have now answered correct on {correctAnswer} out of 5 questions
+          right.
+          <br />
```

Comment:
You should not use <br/> here. Please remove it if it not serving a special purpose.

Reply:
was deleted